packages/usa-file-input/src/index.js

```diff
@@ -210,11 +241,70 @@ const removeOldPreviews = (dropTarget, instructions, inputAriaLabel) => {
     if (instructions) {
       instructions.classList.remove(HIDDEN_CLASS);
     }
-    fileInputElement.setAttribute("aria-label", inputAriaLabel);
     Array.prototype.forEach.call(filePreviews, removeImages);
   }
 };
 
+/**
+ * Update the screen reader-only status message after interaction
+ * @param {HTMLElement} statusElement - screen-reader only container for file status updates
+ * @param {HTMLElement} fileNames - uploaded files
+ * @param {HTMLElement} fileStore - array of uploaded file names
+ */
+const updateStatusMessage = (statusElement, fileNames, fileStore) => {
+  const defaultStatusMessage = statusElement.dataset.defaultStatusText;
+  const statusEl = statusElement;
+  let statusMessage = defaultStatusMessage;
+
+  // If files added, update the status message with file name(s)
+  if (fileNames.length === 1) {
+    statusMessage = `You have selected the file: ${fileStore}`;
+  } else if (fileNames.length > 1) {
+    statusMessage = `You have selected ${
+      fileNames.length
+    } files: ${fileStore.join(", ")}`;
+  }
+
+  // Add delay to encourage screen reader readout
+  setTimeout(() => {
+    statusEl.textContent = statusMessage;
+  }, 1000);
+};
+
+/**
+ * Hide the initial instructions and show the preview heading
+ * @param {HTMLElement} fileInputEl - file input element
+ * @param {HTMLElement} fileNames - uploaded files
+ * @param {HTMLElement} dropTarget - target area div that encases the input
+ * @param {HTMLElement} instructions - text to inform users to drag or select files
+ */
+const addPreviewHeading = (
+  fileInputEl,
+  fileNames,
+  dropTarget,
+  instructions
```

Comment:
Are all these params needed?

Reply:
Simplied params in [866d335](https://github.com/uswds/uswds/pull/5213/commits/866d335ef99f0a712551bb31e44901211006f6c6)